scripts/api.js

```diff
@@ -11,29 +11,6 @@ class ModuleApi {
         return volume * game.settings.get(MODULE_ID, SETTING_VOLUME);
     }
 
-    static getMacroVariables(macro) {
```

Comment:
Moved to shared `utils.js` so it can be used by the "Execute Macro" button in the UI